egs2/opencpop/svs1/local/midi-note.scp

```diff
@@ -105,3 +105,4 @@
 23	B0
 22	A#0/Bb0
 21	A0
+0	rest
```

Comment:
is it OK to be included in this PR?

Reply:
I added a `0 rest` entry to simplify the MIDI mapping. The original opencpop recipe handles 'rest' by mapping it to 0 (`midis = [midi_mapping[note] if note != 'rest' else 0 for note in midis]`). Including `0 rest` directly in the mapping file makes audio segmentation of the ACESinger easier, especially when the transcription includes 'rest' notations. I believe in this way, we can eliminate the need for special handling in the code.

Reply:
Got it.
Thanks for the note.